### ISSUE TITLE
Linking fails on react-native 0.49.3

### ISSUE THREAD

**Issue by ggozad** [rebeccahughes/react-native-device-info#251]
After installing and linking react-native-device-info I get the following error with an upgraded react-native

```
ld: library not found for -lRNDeviceInfo-tvOS
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I guess this is because of the automatic tvOS inclusion on rn. Any workarounds or ways to fix?

**Comment by baijunjie** [comment]
Same question

**Comment by cristianosantos95** [comment]
- open xcode
- project target
- build phases
- link binary with libraries
- remove libRNDeviceInfo-tvOS.a

it worked for me :)

**Comment by cyzaoj** [comment]
What time the bug version

**Comment by XeeshanAnsari** [comment]
remove libRNDeviceinfo-tvOS.a
it also worked for me.

**Comment by lindseymenges** [comment]
I've been banging my head against the wall all day with this issue, and removing libRNDeviceinfo-tvOS.a finally did the trick. Thanks @cristianosantos95 !

**Comment by GantMan** [comment]
Anyone want to see if they can code this fix?

**Comment by machour** [comment]
@GantMan Seems to be handled in this PR #283

**Comment by GantMan** [comment]
Merged!
@machour - any suggested PRs / cleanup before I release an updated version?

**Comment by machour** [comment]
re #283: target seems to have changed from 7.0 to 9.0, not sure that's a good thing (not an iOS expert by any means)
#280: seems harmless enough to me to be merged
#297: needs to make it to README.md

and last but not least, the missing ChangeLogs (including the one for the pending release)

❤️

**Comment by GantMan** [comment]
plz PR the changelogs!  I'll merge and then do a release

**Comment by machour** [comment]
Et voilà : #301
Don't forget to bump the version in here as well: https://github.com/rebeccahughes/react-native-device-info/blob/master/RNDeviceInfo.podspec

**Comment by machour** [comment]
This should now work fine in 0.14.0

**Comment by machour** [comment]
FYI, issue isn't solved in 0.14.0, but is already tracked in #258. Leaving this issue closed.

**Comment by riheldo** [comment]
Well, my issue was diferent, but remove libRNDeviceinfo-tvOS.a works!
